apps/web/modules/d/[link]/d-type-view.tsx

```diff
@@ -20,13 +19,6 @@ export default function Type({
 }: PageProps) {
   return (
     <main className={getBookerWrapperClasses({ isEmbed: !!isEmbed })}>
-      <BookerSeo
```

Comment:
BookerSeo is not needed anymore as we handle the metadata using app router metadata API (by exporting generateMetadata funtion)